### ISSUE TITLE
Unencapsulated / Unusable CSS

### ISSUE THREAD

**Issue by snowman-repos** [suyati/line-control#9]
Integrating the `editor.css` into an existing project is likely to cause a lot of problems because the CSS file overwrites lots of bootstrap and commonly-named classes

**Comment by Mortivor** [comment]
indeed, I had problem with it too. I will have a look into it as soon as I have time.

**Comment by abalakrishnan** [comment]
Yes you are correct. Here an immediate work around is to use your container name before the css rule
